src/index.html

```diff
@@ -11,6 +11,41 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <section class="form">
+      <div class="form__inner">
+        <form
+          action="GET"
+          class="form__big"
+          data-qa="big"
+          method="/"
+        >
+          <label for="big-form" class="form__big-label">
+            <input
+              class="form__big-input"
+              type="search"
+              name="City-small"
+              id="big-form"
+              placeholder="Try &quot;Los Angeles&quot;"
+            >
+          </label>
+        </form>
+        <form
```

Comment:
Don't forget to add empty lines between multiline sibling blocks of HTML
```suggestion
        </form>
        
        <form
```